frost-p256/src/lib.rs

```diff
@@ -0,0 +1,275 @@
+#![allow(non_snake_case)]
+#![deny(missing_docs)]
+#![doc = include_str!("../README.md")]
+
+use elliptic_curve::hash2curve::{hash_to_field, ExpandMsgXmd};
+use hex::FromHex;
+use p256::{
+    elliptic_curve::{
```

Comment:
Do we need to pull in `elliptic-curve` in the crate dependencies if this is exported by `p256`?

Reply:
Good point, removed in fa4f4242c01b6e5031dfa1f6e188ec4604cfab64